apps/music_box_core.F90

```diff
@@ -95,6 +98,7 @@ module music_box_core
   !! Loads input data and initializes model components.
   function constructor( config_file_path ) result( new_obj )
```

Comment:
I wonder if there is a better name than new_obj.

Reply:
probably - any suggestions?